test/sshd/test.sh

```diff
@@ -5,7 +5,9 @@ set -e
 # Optional: Import test library
 source dev-container-features-test-lib
 
-echo 'NO TESTS FOR ME YET!'
+check "shh-init-exists" bash -c "ls /usr/local/share | grep ssh-init.sh"
+check "shhd-log-exists" bash -c "ls /tmp | grep sshd.log"
+check "shhd-log-contents" bash -c "cat /tmp/sshd.log | grep 'Starting OpenBSD Secure Shell server sshd'"
 
```

Comment:
Can we make sure the `sshd` process is running? Something like `ps -aux | grep -v grep | grep sshd`. Note that we need the `| grep -v grep` to remove the grep processes from the `ps` output, otherwise we get false positive matches. There's actually a bug in some of the [other test assertions ](https://github.com/search?q=repo%3Adevcontainers%2Ffeatures+%22ps+-ax%22+path%3Atest%2F**%2F*.sh&type=code) that don't do this.

Reply:
Added check & fixed the 🐛  ones